src/core/qgsfeaturerequest.h

```diff
@@ -704,6 +714,27 @@ class CORE_EXPORT QgsFeatureRequest
      */
     QgsFeatureRequest &setRequestMayBeNested( bool requestMayBeNested );
 
+    /**
+     * Sets the FilterCombination to use when handling filter, default behavior is UseLast.
+     *
+     * \since QGIS 3.12
+     */
+    void setFilterHandling( FilterCombination handlingMethod );
+
+    /**
+     * Returns the filter handling method
+     *
+     * \since QGIS 3.12
+     */
+    QString filterHandling() const;
```

Comment:
This should return a FilterCombination enum value, not a string

Reply:
Ok I'll revert this change, I simply inteded to use the output string as a variable to replace the 'AND' string in the sql expression to allow using OR but I guess an 'if' should be sufficient.